ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.html

```diff
@@ -1 +1,112 @@
+                     [alt]="'App.HomePage.ParticipateSection.' + step.key + '.Title' | translate">
+
+                <figcaption class="step__title"
+                            [innerText]="'App.HomePage.ParticipateSection.' + step.key + '.Title' | translate">
+                </figcaption>
+                <span class="step__time" *ngIf="!last"
+                      [innerText]="'App.HomePage.ParticipateSection.' + step.key + '.Time' | translate">
+                </span>
+                <div class="step__description"
+                     [innerText]="'App.HomePage.ParticipateSection.' + step.key + '.Description' | translate">
+                </div>
+            </figure>
+        </div>
+        <div class="container">
+            <a mat-raised-button
+               [routerLink]="'/' + AppRoutes.AboutUs"
+               class="btn learn-more-btn center"
+               [innerText]="'App.HomePage.ParticipateSection.LearnMoreAboutParticipationButton' | translate">
+            </a>
+        </div>
+    </section>
+
+    <section class="faq">
+        <h2 class="title" translate>App.FAQ.Title</h2>
+        <div class="container">
+            <app-faq-section questions="App.FAQ.Questions" questionsCount="5"></app-faq-section>
+        </div>
+        <a mat-raised-button
+           [routerLink]="'/' + AppRoutes.FAQ"
+           class="btn more-faq-btn center"
+           [innerText]="'App.FAQ.Button.MoreFAQs' | translate">
+        </a>
+    </section>
+
+    <section class="stay-informed">
+        <div class="container">
+            <h2 class="title" translate>App.HomePage.StayInformedSection.Title</h2>
+            <div class="social-media">
+                <div class="text" translate>App.HomePage.StayInformedSection.Text</div>
+                <div class="social-media__links">
+                    <a class="social-media__link" [href]="twitterUrl" target="_blank">
+                        <img class="social-media__logo" lazy-resource src="assets/images/twitter.svg"
+                             [alt]="'Common.Alts.Twitter' | translate">
+                    </a>
+                    <a class="social-media__link" [href]="facebookUrl" target="_blank">
+                        <img class="social-media__logo" lazy-resource src="assets/images/facebook.svg"
+                             [alt]="'Common.Alts.Facebook' | translate">
+                    </a>
+                    <a class="social-media__link" [href]="instagramUrl" target="_blank">
+                        <img class="social-media__logo" lazy-resource src="assets/images/instagram.svg"
+                             [alt]="'Common.Alts.Instagram' | translate">
+                    </a>
+                </div>
+            </div>
+            <a mat-raised-button
+               class="btn join-btn"
+               [innerText]="'App.HomePage.StayInformedSection.JoinMailingListButton' | translate">
+            </a>
```

Comment:
do we have a separate ticket for this?

Reply:
I guess we don't have extra tickets for PanCan yet (except for static pages).

Reply:
so Join Mailing List is not a part of homepage? should we create a ticket for it?

Reply:
Created [DDP-6401](https://broadinstitute.atlassian.net/browse/DDP-6401)